### PR TITLE
feat: add db alias

### LINKED ISSUE #18: initialize_enforcer with other DB_ALIAS
if django database config in `backend/settings.py` like:
---
DATABASES = {
    'default': {...},
    'acount': {...},
}
---
and we want use `account` db to store casbin_rule, HOW?

### PR DESCRIPTION
**Pull request by BustDot**
Add CASBIN_DB_ALIAS argument to configure db.
fix: https://github.com/pycasbin/django-orm-adapter/issues/18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,9 @@ E.g. if you wish to use the file adapter
 set the adapter to `casbin.persist.adapters.FileAdapter` and use
 `CASBIN_ADAPTER_ARGS = ('path/to/policy_file.csv',)`
 
+### `CASBIN_DB_ALIAS`
```
Comment (hsluoyz) [inline README.md:83]: Use `DB_NAME`

File: casbin_adapter/adapter.py [hunk 2]
```diff
@@ -11,10 +11,13 @@
...
     """the interface for Casbin adapters."""
 
+    def __init__(self, db_alias="default"):
+        self.db_alias = db_alias
+
     def load_policy(self, model):
         """loads all policy rules from the storage."""
         try:
-            lines = CasbinRule.objects.all()
+            lines = CasbinRule.objects.using(self.db_alias).all()
```
Comment (hsluoyz) [inline casbin_adapter/adapter.py:20]: self.db_name

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@hefeiyun plz review

File: casbin_adapter/adapter.py (hunk 2 above)
Comment (BustDot) [reply casbin_adapter/adapter.py:20]: In django, all database operations are done by specifying a database alias in settings.py instead of database name.

Comment (hsluoyz) [reply casbin_adapter/adapter.py:20]: OK

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**